tests/run-pass/stacked-borrows/2phase.rs

```diff
@@ -26,7 +28,21 @@ fn two_phase3(b: bool) {
     ));
 }
 
-/*
+#[allow(unreachable_code)]
+fn two_phase_raw() {
+    let x: &mut Vec<i32> = &mut vec![];
+    x.push(
+        {
+            // Unfortunately this does not trigger the problem of creating a
+            // raw ponter from a pointer that had a two-phase borrow derived from
+            // it because of the implicit &mut reborrow.
+            let raw = x as *mut _;
```

Comment:
will `{x}` skip that reborrow? Or just move the reborrow to a temporary?

Reply:
It'll replace the reborrow by some reborrows and a move. The move then also becomes a reborrow because of retagging, but that doesn't even make a different any more.